### ISSUE TITLE
关于BDD100K的label问题

### ISSUE THREAD

**Issue by zhangjy2008327** [cardwing/Codes-for-Lane-Detection#233]
请问，BDD100K是个2分类还是个5分类问题？从数据库来看，应该是个5分类问题？可是，从提供的ground truth图片的值来看，ground truth中只有255 和0的值，并且其ground truth的宽度也不一样(train.png的宽度是多少？ val.png的宽度是多少？)，这应该不是多分类的label?请问，在训练时如何使用ground truth来进行5分类训练？谢谢，请有时间回复一下。

**Comment by cardwing** [comment]
ICCV论文里的实现细节有提到怎么处理BDD100K的数据。“ As to BDD100K, we resize the image to
360×640 to save memory usage. The lanes of BDD100K are labelled by two lines. Training the networks using the provided labels is tricky. Therefore, based on these two lines, we calculate the center lines as new targets. We dilate ground-truth lanes of the training set of BDD100K as 8 pixels to provide denser targets while keeping these of testing set unchanged (2 pixels).”
